test/cqlpy/test_tools.py

```diff
@@ -1731,3 +1731,15 @@ def test_scylla_sstable_query_temp_dir(cql, scylla_path, scylla_data_dir):
         assert res.returncode == 2
         assert res.stdout == ""
         assert res.stderr.endswith(f"error running operation: std::filesystem::__cxx11::filesystem_error (error generic:20, filesystem error: temp_directory_path: Not a directory [{f.name}])\n")
+
+
+def test_scylla_sstable_query_null_data(cql, test_keyspace, scylla_path, scylla_data_dir):
```

Comment:
Please add a comment explaining what this test reproduces (including mentioning the issue number). It's impossible to guess. I have no idea what "null data" refers to, or what was the error mode before it was fixed. Because I see your test doesn't actually check the content of the output - just that it didn't fail, right?